src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -223,14 +208,14 @@ export class ColorPickerHexInput implements LoadableComponent {
   //--------------------------------------------------------------------------
 
   render(): VNode {
-    const { intlHex, value } = this;
+    const { value } = this;
     const hexInputValue = this.formatForInternalInput(value);
 
     return (
       <div class={CSS.container}>
         <calcite-input
           class={CSS.input}
-          label={intlHex}
+          label={"hex"}
```

Comment:
Does this ever need to be translated?

Reply:
this needs to be translated. one way is to parse it from the `color-picker` component as `messages` similar to what we did for `date-picker-month-header`.